cmd/webhook_v2.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/gobuffalo/flect"
+	"github.com/spf13/cobra"
+
+	"sigs.k8s.io/kubebuilder/pkg/scaffold"
+	"sigs.k8s.io/kubebuilder/pkg/scaffold/input"
+	"sigs.k8s.io/kubebuilder/pkg/scaffold/project"
+	"sigs.k8s.io/kubebuilder/pkg/scaffold/v1/resource"
+	resourcev2 "sigs.k8s.io/kubebuilder/pkg/scaffold/v2"
+	"sigs.k8s.io/kubebuilder/pkg/scaffold/v2/webhook"
+)
+
+func newWebhookV2Cmd() *cobra.Command {
+	o := webhookV2Options{}
+
+	cmd := &cobra.Command{
+		Use:   "webhook",
+		Short: "Scaffold a webhook using v2 scaffolding.",
```

Comment:
We don't have to reveal `v1/v2` to the end-users. So we may be we can just word it:
`Scaffold a webhook for an API or resource ?`